CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
   `:authority` value for UDS connections to `localhost` instead of using
   UDS path. Using path triggers checks in HTTP2 servers which
   attempt to validate `:authority` value.
+* `ChannelOptions` now exposes `connectTimeout`, which is used on the 
```

Comment:
This is in the wrong section. It needs to go into `## 3.1.1-dev`.

Reply:
Done